web/styles/input_pill.css

```diff
@@ -37,6 +37,9 @@
         }
 
         .pill-value {
+            /* Treat as flex container, with an anonymous block around
+              the username text node, for positioning status emoji */
+            display: flex;
```

Comment:
Have you done an audit of the other places that we use input-pill elements? I believe we use them in the search typeahead results, when picking users in stream settings, and a few other places.